molecule/test11/molecule.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: single partition, group is partition
+driver:
+  name: docker
+platforms:
+  - name: testohpc-login-0
+    image: ${MOLECULE_IMAGE}
+    pre_build_image: true
+    groups:
+      - testohpc_login
+    command: /sbin/init
+    tmpfs:
+      - /run
+      - /tmp
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    networks:
+      - name: net1
+  - name: testohpc-compute-0
+    image: ${MOLECULE_IMAGE}
+    pre_build_image: true
+    groups:
+      - testohpc_compute
+      - testohpc_compute_orig
```

Comment:
Is this used now?

Reply:
Missed that `compute_orig` is in converge.yml

Reply:
Yes. Test 11's `converge.yml` defines the partition as `compute_orig`, which uses that group. I couldn't find a way to modify the group so actually it changes what the single defined partition is.